src/macros.rs

```diff
@@ -0,0 +1,13 @@
+#[macro_export]
+macro_rules! assert_diff_eq {
+    ($actual:expr, $expected:expr) => {{
+        use regex::Regex;
+        use std::str;
+
+        let diff = str::from_utf8(&$actual).unwrap();
+        let re = Regex::new(r"\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}\.\d+ [+-]\d{4}").unwrap();
+        let actual = re.replace_all(diff, "");
```

Comment:
That looks good! For added robustness, could you replace the matches of the regex with a hardcoded placeholder value (e.g. "TIMESTAMP"). This would require updating the expected values in several tests to include that placeholder.

Also, can you do the replacement only on the first two lines of the diff?
Even though this is used only in tests where we supposedly control the contents of the test data, we don't want to be replacing timestamps in the actual diff by accident.